susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -131,6 +131,7 @@ trust_suse_manager_tools_deb_gpg_key:
 salt-minion-package:
   pkg.latest:
     - name: salt-minion
+    - install_recommends: False
```

Comment:
NOTE: This only works for apt-based systems (see https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html)